action.yml

```diff
@@ -48,6 +48,11 @@ runs:
         GH_TOKEN: ${{ inputs.token }}
       shell: bash
       run: |
+        if [ "$GH_TOKEN" = "" ]; then
+          echo "Error! For destination=default you have to pass token input also."
+          exit 1
+        fi
+
         if ! type gh > /dev/null; then
           curl -L https://github.com/cli/cli/releases/download/v2.24.3/gh_2.24.3_linux_amd64.tar.gz | tar xz --strip-components=2 -C /usr/local/bin/ gh_2.24.3_linux_amd64/bin/gh
```

Comment:
Please move the remote bin path to an environment variable?

Reply:
I think I will move gh setup to separate action

Reply:
Done in https://github.com/metanorma/ci/pull/110